src/App.vue

```diff
@@ -16,12 +16,15 @@ import Welcome from './components/Welcome.vue';
 import Footer from './components/Footer.vue';
 
 export default {
-  name: 'app',
   components: {
+    // See: https://github.com/vuejs/eslint-plugin-vue/issues/556
+    // eslint-disable-next-line vue/no-unused-components
```

Comment:
So despite this issue still existing the Vue Devs closed the Issue?

Reply:
It's fixed but released in the next version